vulnerability-rating-taxonomy.json

```diff
@@ -1835,7 +1873,99 @@
               "priority": 2
             }
           ]
-        }
+        },
+         {
+          "id": "weakness_in_firmware_updates",
```

Comment:
@TimmyBugcrowd taxonomy mentioned in the PR descriptio doesn't match with the JSON. 
I can see that 'Command Injection' is added as a subcategory but under it there are no children, instead all expected children are added separately under "Insecure OS/Firmware" itself.
Is this expected ?

Here are the line items that I was able to observe
Insecure OS/Firmware - Command Injection - Weakness in Firmware Updates - Firmware cannot be updated - VARIES
Insecure OS/Firmware - Command Injection - Weakness in Firmware Updates - Firmware does not validate update integrity- P3
Insecure OS/Firmware - Command Injection - Weakness in Firmware Updates - Firmware is not encrypted- P5

Insecure OS/Firmware - Command Injection - Kiosk Escape or Breakout - VARIES
Insecure OS/Firmware - Command Injection - Poorly Configured Disk Encryption - VARIES
Insecure OS/Firmware - Command Injection - Shared Credentials on Storage - P3
Insecure OS/Firmware - Command Injection - Over-Permissioned Credentials on Storage - P2
Insecure OS/Firmware - Command Injection - Local Administrator on default environment - P2
Insecure OS/Firmware - Command Injection - Poorly Configured Operating System Security - VARIES
Insecure OS/Firmware - Command Injection - Recovery of Disk Contains Sensitive Material - VARIES
Insecure OS/Firmware - Command Injection - Failure to Remove Sensitive Artifacts from Disk - VARIES

Reply:
That is expected. I just corrected the description above. Nice catch and thank you!